js/components/plp-r-code.html

```diff
@@ -235,10 +238,14 @@
     # You can save and load the full results structure using:
     # PatientLevelPrediction::savePlpResult(results, dirPath = file.path(getwd(), "lr"))
     # results <span class="token operator"><-</span> PatientLevelPrediction::loadPlpResult(file.path(getwd(), "lr"))</span>
-		
+
+    <span class="token comment">#to interactively view the performance</span>
+    PatientLevelPrediction::viewPlp(results) 
+
+
     <span class="token comment"># add plots and document to output folder</span>
     PatientLevelPrediction::plotPlp(results, 
-                                         file.path(getwd(),'plpmodels', results$analysisRef$analysisId))
```

Comment:
I don't know if you're just trying to mess with me with this formatting change... :)

Reply:
Who me? Never :-) This small change is to fix the formatting for display in the UI for the code - the function parameters were not aligned properly.